README.md

```diff
@@ -82,7 +82,10 @@ Example:
 ```js
 'use strict';
 const Model = require('screwdriver-models');
-const factory = Model.PipelineFactory.getInstance({ datastore });
+const factory = Model.PipelineFactory.getInstance({
+    datastore,
+    scmPlugin,          // scm plugin.  For example, screwdriver-scm-github
```

Comment:
I would need to add how to instantiate scm-github correctly, that's why I didn't want to do it.